tests/test_on_data/test_imaging_interfaces.py

```diff
@@ -71,6 +77,142 @@ def check_extracted_metadata(self, metadata: dict):
         )
 
 
+@parameterized_class(
+    [
+        {
+            "interface_kwargs": {
+                "folder_path": str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage"),
+                "file_pattern": "scanimage_20240320_multifile*.tif",
+                "channel_name": "Channel 1",
+            }
+        },
+        {
+            "interface_kwargs": {
+                "folder_path": str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage"),
+                "file_pattern": "scanimage_20240320_multifile*.tif",
+                "channel_name": "Channel 2",
+            }
+        },
+    ]
+)
+class TestScanImageSinglePlaneMultiFileImagingInterface(ScanImageSinglePlaneMultiFileImagingInterfaceMixin, TestCase):
+    data_interface_cls = ScanImageSinglePlaneMultiFileImagingInterface
+    save_directory = OUTPUT_PATH
```

Comment:
```suggestion
@parameterized_class(
    [
        {"channel_name": "Channel 1"},
        {"channel_name": "Channel 2"},
    ],
)
class TestScanImageSinglePlaneMultiFileImagingInterface(ScanImageSinglePlaneMultiFileImagingInterfaceMixin, TestCase):
    data_interface_cls = ScanImageSinglePlaneMultiFileImagingInterface
    save_directory = OUTPUT_PATH
    interface_kwargs = dict(
        folder_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage"),
        file_pattern="scanimage_20240320_multifile*.tif",
    )
    channel_name = "Channel 1"

    def __init__(self, *args, **kwargs) -> None:
        self.interface_kwargs["channel_name"] = self.channel_name
        super().__init__(*args, **kwargs)
```

Reply:
This is how I would suggest to fix the problem.

Reply:
Thank you for this suggestion!